diff --git a/src/obs-ndi.cpp b/src/obs-ndi.cpp
index 283d64f..986e848 100644
--- a/src/obs-ndi.cpp
+++ b/src/obs-ndi.cpp
@@ -230,8 +230,7 @@ const NDIlib_v4 *load_ndilib()
 		locations << path;
 	}
 #if defined(__linux__) || defined(__APPLE__)
-	locations << "/usr/lib";
-	locations << "/usr/local/lib";
+	locations << "/@NDI@/lib";
 #endif
 	for (QString location : locations) {
 		path = QDir::cleanPath(
